nuxeo-features/nuxeo-thumbnail/src/main/resources/OSGI-INF/thumbnailing-bulk-contrib.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.thumbnailing.bulk" version="1.0.0">
+
+  <require>org.nuxeo.ecm.core.bulk</require>
+
+  <extension target="org.nuxeo.ecm.core.bulk" point="actions">
+    <action name="recomputeThumbnails" bucketSize="100" batchSize="50" httpEnabled="false"
```

Comment:
Building thumbnail is slow it requires to invoke external process, I prefer to limit the batch transaction, so a better default could be:
bucketSize="100" batchSize="10"